otx/algorithms/common/configs/training_base.py

```diff
@@ -190,6 +190,15 @@ class BaseLearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING,
         )
 
+        auto_decrease_batch_size = configurable_boolean(
+            default_value=False,
+            header="Decrease batch size if current batch size isn't fit to CUDA memory.",
+            description="Find a proper batch size by training for an iteration with various batch size a few times.",
+            warning="Checking a current batch size is available and adapting it if necessary "
```

Comment:
I cannot understand the sentence. Additionally, it would be better to include some points like

- Actual batch size might be lower than default or user's input
- Learning rate might also be adjusted according to batch size reduction
- It might take a little bit extra time.

Reply:
Thanks for detailed explanation. I updated all of warning comment as you said. Please take a look.